src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -342,15 +343,20 @@ private void InitializeLinuxLogging()
                 // The logging service for linux works by capturing EventSource messages,
                 // which our linux platform does not recognize, and logging them via a
                 // different strategy such as writing to console or to a file.
-                this.eventSourceListener = new EventSourceListener(linuxLogger);
+
+                // Since our logging payload can be quite large, linux telemetry by default
+                // disables verbose-level telemetry to avoid a performance hit.
+                bool enableVerbose = this.Options.Tracing.AllowVerboseLinuxTelemetry;
+                this.eventSourceListener = new EventSourceListener(linuxLogger, enableVerbose);

```

Comment:
Added a new tracing option to let customers enable verbose telemetry in linux, disabled by default